transports/ssl/src/main/resources/META-INF/mule-tls.xsd

```diff
@@ -137,23 +137,43 @@
         </xsd:annotation>
     </xsd:element>
 
+    <xsd:element name="standard-revocation-check" type="tlsContextStandardRevocationCheckType">
+        <xsd:annotation>
+            <xsd:documentation>
+                The standard certificate revocation check provided by the JVM. It reads the OCSP and CRLDP fields in each certificate (when present and enabled) and validates accordingly.
+            </xsd:documentation>
+        </xsd:annotation>
+    </xsd:element>
+
+    <xsd:element name="crl-file-revocation-check" type="tlsContextCrlFileRevocationCheckType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Custom certificate validation through a local path to a CRL (ignores CRLDP or OCSP mentioned in certificates)..
+            </xsd:documentation>
+        </xsd:annotation>
+    </xsd:element>
+
+    <xsd:element name="custom-ocsp-revocation-check" type="tlsContextCustomOcspRevocationCheckType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Custom certificate validation trough a specific OCSP server (ignores CRLDP or OCSP mentioned in certificates).
+            </xsd:documentation>
+        </xsd:annotation>
+    </xsd:element>
+
     <xsd:complexType name="tlsContextType">
         <xsd:complexContent>
             <xsd:extension base="mule:abstractExtensionType">
-                <xsd:choice minOccurs="1">
-                    <xsd:sequence>
-                        <xsd:element ref="trust-store" minOccurs="1" maxOccurs="1" />
-                        <xsd:element ref="key-store" minOccurs="0" maxOccurs="1" />
-                    </xsd:sequence>
-                    <xsd:sequence>
-                        <xsd:element ref="trust-store" minOccurs="0" maxOccurs="1" />
-                        <xsd:element ref="key-store" minOccurs="1" maxOccurs="1" />
-                    </xsd:sequence>
-                    <xsd:sequence>
-                        <xsd:element ref="trust-store" minOccurs="1" maxOccurs="1" />
-                        <xsd:element ref="key-store" minOccurs="1" maxOccurs="1" />
-                    </xsd:sequence>
-                </xsd:choice>
+                <xsd:sequence minOccurs="1">
+                    <xsd:element ref="trust-store" minOccurs="0" maxOccurs="1" />
```

Comment:
I'm not sure about this. The other config was to support having nothing, just one of each or all definitions. Please review this with @pablolagreca because there was a reason for having that separate.